scenario/LiquidationBotScenario.ts

```diff
@@ -137,13 +292,382 @@ for (let i = 0; i < MAX_ASSETS; i++) {
         poolFee: daiPool.poolFee
       });
 
+      // confirm that Albert position has been abosrbed
       expect(await comet.isLiquidatable(albert.address)).to.be.false;
       expect(await comet.collateralBalanceOf(albert.address, collateralAssetAddress)).to.eq(0);
 
-      // XXX confirm that protocol reserves have increased
-
       // check that recipient balance increased
       expect(await USDC.balanceOf(RECIPIENT)).to.be.greaterThan(Number(initialRecipientBalance));
     }
   );
-}
\ No newline at end of file
+}
+
+scenario(
+  `LiquidationBot > absorbs, but does not attempt to purchase collateral when value is beneath liquidationThreshold`,
+  {
+    filter: async (ctx) => ctx.world.base.network === 'mainnet',
+    tokenBalances: {
+      $comet: { $base: 1000 },
+    },
+    cometBalances: {
+      albert: {
+        $asset0: ' == 5',
+      },
+      betty: { $base: 1000 },
+    },
+  },
+  async ({ comet, actors, assets }, _context, world) => {
+    const { albert, betty } = actors;
+    const { USDC } = assets;
+
+    const liquidator = await world.deploymentManager.deploy(
+      'liquidator',
+      'liquidator/Liquidator.sol',
+      [
+        RECIPIENT, // _recipient
+        UNISWAP_ROUTER, // _uniswapRouter
+        SUSHISWAP_ROUTER, // _sushiSwapRouter
+        comet.address, // _comet
+        UNISWAP_V3_FACTORY_ADDRESS, // _factory
+        WETH9, // _WETH9
+        1000e6, // _liquidationThreshold,
+        [
+          '0xc00e94Cb662C3520282E6f5717214004A7f26888', // COMP
+          '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599', // WBTC
+          '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', // WETH
+          '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984', // UNI
+          '0x514910771AF9Ca656af840dff83E8264EcF986CA', // LINK
+        ],
+        [
+          true,
+          true,
+          false,
+          true,
+          true
+        ],
+        [
+          3000,
+          3000,
+          500,
+          3000,
+          3000
+        ],
+        [
+          Exchange.SushiSwap, // COMP
+          Exchange.Uniswap,   // WBTC
+          Exchange.Uniswap,   // WETH
+          Exchange.Uniswap,   // UNI
+          Exchange.Uniswap,   // LINK
+        ],
+        [
+          constants.MaxUint256, // COMP
+          constants.MaxUint256, // WBTC
+          constants.MaxUint256, // WETH
+          constants.MaxUint256, // UNI
+          constants.MaxUint256, // LINK
+        ]
+      ]
+    );
```

Comment:
Could use a helper function for this since it's being called in every scenario.

Reply:
I guess it might be slightly hard to define one since we're tweaking certain parts of the config in each scenario

Reply:
yeah, it would only save us a few lines that don't change across tests